src/Bridge/Doctrine/Orm/Util/IdentifierManagerTrait.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ApiPlatform\Core\Bridge\Doctrine\Orm\Util;
+
+use ApiPlatform\Core\Exception\PropertyNotFoundException;
+use Doctrine\Common\Persistence\ObjectManager;
+
+/**
+ * @internal
+ */
+trait IdentifierManagerTrait
```

Comment:
I still think `IdentifierManagerTrait` is a bad name. `CompositeIdentifierTrait` makes sense to me, because this is a trait that helps with handling composite identifiers (even when the identifier is not composite).

Or perhaps another, because the "manager" part of `IdentifierManagerTrait` definitely does not inspire confidence that it is not violating SRP (a manager trait?!).